src/main/java/domain/Domain.java

```diff
@@ -2,18 +2,19 @@
 
 import controller.BusController;
 import controller.DriverController;
+import controller.LocationController;
 
 public class Domain {
 
 	public static void main(String[] args) {
 
 		DriverController driverController = new DriverController();
 		BusController busController = new BusController();
+		LocationController locationController = new LocationController();
 
 		driverController.listAllDrivers();
 		busController.listAllBuses();
-		System.out.println("Let's add new Driver");
-		driverController.create("Ivanka Trump", "Tall building", "+1002555",25,174);
-		driverController.listAllDrivers();
```

Comment:
i think because u have deleted these lines after you did merge, they will be deleted from master branch for all of us.
We need to find solution how to avoid clones creations without cleaning the Domain class